README.md

```diff
@@ -55,6 +74,7 @@ This section should list any major frameworks that you built your project using.
 
 ### Prerequisites
```

Comment:
Change Prerequisites to installation
and remove the title ** installation** from line 73